### ISSUE TITLE
Debug logging on destroy very chatty with e.g. ColumnHeader LocalModel

### ISSUE THREAD

**Issue by amcclain** [xh/hoist-react#1427]
Disclaimer: this is definitely a nit, but ends up looking kinda of wrong to me on the console (and might point to some other underlying issue / unoptimized flow). 

If you load up e.g. Toolbox standard grid with console debug logging enabled, you will see a flood of "Destroying" log entries as the grid (I believe) re-renders its column headers, and each render destroys/creates a local model instance.

I am wondering a) if we expect this rapid/initial model swapping for the grid headers in particular and b) if we want to keep logging _all_ destructions on debug.  (Where's `console.trace()` when you need it?)

**Comment by lbwexler** [comment]
I agree this is wrong in a couple of ways.

First of all, we should go ahead and stop logging the destructions by default.  It's way too chatty.  Its a trivial piece of code to add back if we ever need it.  

The dream has always been to have package specific logging -- maybe we will get there.

Having said all that, it is weird that those headers are getting **re-created/re-mounted** on initialization.  That definitely seems like something sub-optimal.  To be clear -- its not *re-rendering*  if that were the case, the local model would not be destroyed.

**Comment by lbwexler** [comment]
I went ahead and convince myself that this is really nothing obvious we are doing wrong here:

We are indeed only rendering the grid once, setting these column definitions once, and not calling any other API method to reset the column defs.  Also, made the headers dumb, and not reactive, and still seeing the header components being remounted.    So I think probably an inefficiency in the ag-Grid code.  Good for us that we are cleaning up reactions needed by our Header Component!

Removed the logging line directly on master.
